src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/TabOrder.cs

```diff
@@ -329,9 +301,10 @@ private void DrawTabs(IList tabs, Graphics gr, bool fRegion)
 
             if (fRegion)
             {
-                ctl = (Control)host.RootComponent;
-                rc = GetConvertedBounds(ctl);
-                region.Intersect(rc);
+                Debug.Assert(host is not null, "host is set in the contructor should not be null");
+                Control rootControl = (Control)host.RootComponent;
+                rc = GetConvertedBounds(rootControl);
+                region?.Intersect(rc);
```

Comment:
This can be changed to the following to accomodate for my above comment about moving those variables inside the loop
```suggestion
                region?.Intersect(GetConvertedBounds(rootControl));
```